ext/zeek_pygments.py

```diff
@@ -1,165 +1,247 @@
-                    'rotate_size', 'encrypt', 'raw_output', 'mergeable', 'error_handler',
-                    'broker_allow_complex_type', 'is_assigned', 'is_used',
-                    'type_column', 'deprecated', 'on_change', 'backend', 'broker_store'),
-                prefix=r'&', suffix=r'\b'),
-             Keyword.Pseudo),
+        "attributes": [
+            (
+                words(
+                    (
+                        "redef",
+                        "priority",
+                        "log",
+                        "optional",
+                        "default",
+                        "add_func",
+                        "delete_func",
+                        "expire_func",
+                        "read_expire",
+                        "write_expire",
+                        "create_expire",
+                        "synchronized",
+                        "persistent",
+                        "rotate_interval",
+                        "rotate_size",
+                        "encrypt",
+                        "raw_output",
+                        "mergeable",
+                        "error_handler",
+                        "broker_allow_complex_type",
+                        "is_assigned",
+                        "is_used",
+                        "type_column",
+                        "deprecated",
+                        "on_change",
+                        "backend",
+                        "broker_store",
+                    ),
+                    prefix=r"&",
+                    suffix=r"\b",
+                ),
+                Keyword.Pseudo,
+            ),
         ],
-
-        'types': [
-            (words(('any',
-                    'enum', 'record', 'set', 'table', 'vector',
-                    'function', 'hook', 'event',
-                    'addr', 'bool', 'count', 'double', 'file', 'int', 'interval',
-                    'pattern', 'port', 'string', 'subnet', 'time'),
-                prefix=r'\b', suffix=r'\b'),
-             Keyword.Type),
-
-            (r'\b(opaque)(\s+)(of)(\s+)((?:[A-Za-z_][A-Za-z_0-9]*)(?:::(?:[A-Za-z_][A-Za-z_0-9]*))*)\b',
-                bygroups(Keyword.Type, Text, Operator.Word, Text, Keyword.Type)),
-
-            (r'\b(type)(\s+)((?:[A-Za-z_][A-Za-z_0-9]*)(?:::(?:[A-Za-z_][A-Za-z_0-9]*))*)(\s*)(:)(\s*)\b(record|enum)\b',
-                bygroups(Keyword, Text, Name.Class, Text, Operator, Text, Keyword.Type)),
-
-            (r'\b(type)(\s+)((?:[A-Za-z_][A-Za-z_0-9]*)(?:::(?:[A-Za-z_][A-Za-z_0-9]*))*)(\s*)(:)',
-                bygroups(Keyword, Text, Name, Text, Operator)),
-
-            (r'\b(redef)(\s+)(record|enum)(\s+)((?:[A-Za-z_][A-Za-z_0-9]*)(?:::(?:[A-Za-z_][A-Za-z_0-9]*))*)\b',
-                bygroups(Keyword, Text, Keyword.Type, Text, Name.Class)),
+        "types": [
+            (
+                words(
+                    (
+                        "any",
```

Comment:
This is one of the rare places where the earlier form was slightly more meaningful because it grouped by context within a line. But, no biggie.